sal_interfaces/Watcher/Watcher_Events.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+    <Enumeration>AlarmSeverity_None,AlarmSeverity_Warning,AlarmSeverity_Serious,AlarmSeverity_Critical</Enumeration>
+
+<SALEvent>
+    <Subsystem>Watcher</Subsystem>
+    <Version>1.0.0</Version>
+    <Author></Author>
+    <EFDB_Topic>Watcher_logevent_heartbeat</EFDB_Topic>
+    <Alias>heartbeat</Alias>
+    <Explanation>http://sal.lsst.org</Explanation>
+    <item>
+        <EFDB_Name>heartbeat</EFDB_Name>
+        <Description></Description>
+        <IDL_Type>boolean</IDL_Type>
+        <Units></Units>
+        <Count>1</Count>
+    </item>
+</SALEvent>
+
+<SALEvent>

```

Comment:
This looks pretty good - we may want the condition that triggered the alarm as described in LTS-897-REQ-0006

Reply:
Interesting point. Let's talk about it. I have no idea what that field might contain.

I am also missing ack timeout and escalation. I think they should be timestamps rather than delta-times, as it eliminates any display errors caused by delay of the sample.

Do we plan to support automatic acknowledgement for any alarms? If so, that timestamp should also be present.

Reply:
These look good.  Per our previous conversation, this should work.